app/assets/stylesheets/index.scss

```diff
@@ -56,3 +56,11 @@
   text-align: center;
   margin-bottom: 58px;
 }
+.alert {
+  margin: 0 auto;
+  color: 	#32CD32;
+  font-size: 15px;
+  background-color: #90EE90;
```

Comment:
Probably, it would be useful to add a variable for `color` and `background-color`